cfgov/jinja2/v1/_includes/molecules/text-introduction.html

```diff
@@ -20,8 +20,10 @@
    ========================================================================== #}
 
 <h1>{{ value.heading }}</h1>
-<div class="lead-paragraph">{{ value.intro.source | safe }}</div>
-{{ value.body.source | safe }}
+<div class="lead-paragraph">{{ value.intro | safe }}</div>
+<p>
+{{ value.body | safe }}
+</p>
```

Comment:
Is there a reason there's a paragraph added here but paragraphs were removed elsewhere?